presto-main/src/test/java/com/facebook/presto/type/TestTime.java

```diff
@@ -37,34 +33,17 @@
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
 import static com.facebook.presto.type.IntervalDayTimeType.INTERVAL_DAY_TIME;
 import static com.facebook.presto.util.DateTimeZoneIndex.getDateTimeZone;
-import static io.airlift.testing.Closeables.closeAllRuntimeException;
 
 public class TestTime
+    extends AbstractTestFunctions
 {
     private static final TimeZoneKey TIME_ZONE_KEY = getTimeZoneKey("Europe/Berlin");
     private static final DateTimeZone DATE_TIME_ZONE = getDateTimeZone(TIME_ZONE_KEY);
 
-    private FunctionAssertions functionAssertions;
-
-    @BeforeClass
-    public void setUp()
-    {
-        Session session = testSessionBuilder()
+    public TestTime() {
```

Comment:
This will fail checkstyle. I would squash all but the first commit a single *"Use AbstractTestFunctions in time related tests"* commit and run the code formatter.